source/views/sis/student-work/detail.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+import React from 'react'
+import {Text, ScrollView, StyleSheet} from 'react-native'
+import {fastGetTrimmedText} from '../../../lib/html'
+import {Cell, Section, TableView} from 'react-native-tableview-simple'
+import moment from 'moment'
+import openUrl from '../../components/open-url'
+import Communications from 'react-native-communications'
+import * as c from '../../components/colors'
+import type {JobType} from './types'
+
+const styles = StyleSheet.create({
+  selectable: {
+    paddingVertical: 10,
+  },
+  lastUpdated: {
+    paddingBottom: 20,
+  },
+  footer: {
+    fontSize: 10,
+    color: c.iosDisabledText,
+    textAlign: 'center',
+  },
+})
+
+function renderTitle(title: string, category: string) {
+  const trimmedTitle = fastGetTrimmedText(title)
+  const trimmedDetail = fastGetTrimmedText(category)
+  return trimmedTitle || trimmedDetail
+    ? <Section header="JOB">
+        <Cell
+          cellStyle="Subtitle"
+          title={trimmedTitle}
+          detail={trimmedDetail}
+        />
+      </Section>
+    : null
+}
+
+function renderContact(
+  title: string,
+  office: string,
+  contact: string,
+  email: string,
+) {
+  const trimmedOffice = fastGetTrimmedText(office)
+  const trimmedContact =
+    fastGetTrimmedText(contact) || fastGetTrimmedText(email)
+  return trimmedOffice || trimmedContact
+    ? <Section header="CONTACT">
+        <Cell
+          cellStyle="Subtitle"
+          title={trimmedContact}
+          detail={trimmedOffice}
+          accessory="DisclosureIndicator"
+          onPress={() => openEmail(email, title)}
+        />
+      </Section>
+    : null
+}
+
+function renderHours(timeOfHours: string, hoursPerWeek: string) {
+  const time = fastGetTrimmedText(timeOfHours)
+  const hours = fastGetTrimmedText(hoursPerWeek)
+  const ending = hours == 'Full-time' ? '' : ' hrs/week'
+  return time && hours
+    ? <Section header="HOURS">
+        <Cell
+          cellStyle="Subtitle"
+          title={time}
+          detail={hoursPerWeek + ending}
+        />
+      </Section>
+    : null
+}
+
+function renderDescription(description: string) {
+  const trimmedDescription = description.replace(/\t/g, ' ')
```

Comment:
Instead of replacing tabs, what about replacing any consecutive whitespace with a single space? This is just a random idea.